src/NzbDrone.Common/Serializer/Json.cs

```diff
@@ -24,22 +26,34 @@ static Json()
 
 
             SerializerSetting.Converters.Add(new StringEnumConverter { CamelCaseText = true });
-            //SerializerSetting.Converters.Add(new IntConverter());
             SerializerSetting.Converters.Add(new VersionConverter());
             SerializerSetting.Converters.Add(new HttpUriConverter());
 
             Serializer = JsonSerializer.Create(SerializerSetting);
 
+
+            DeserializerSetting = new JsonSerializerSettings
+            {
+                DateTimeZoneHandling = DateTimeZoneHandling.Utc,
+                NullValueHandling = NullValueHandling.Include,
+                Formatting = Formatting.Indented,
+                DefaultValueHandling = DefaultValueHandling.Ignore,
+                ContractResolver = new CamelCasePropertyNamesContractResolver()
+            };
+
+
+            DeserializerSetting.Converters.Add(new StringEnumConverter { CamelCaseText = true });
+            DeserializerSetting.Converters.Add(new VersionConverter());
```

Comment:
Add DeserializerSetting.Converters.Add(new HttpUriConverter());

or extract adding the converters to a separate method.